docker/daemon.go

```diff
@@ -413,7 +412,7 @@ func initRouter(s *apiserver.Server, d *daemon.Daemon) {
 		image.NewRouter(d, decoder),
 		systemrouter.NewRouter(d),
 		volume.NewRouter(d),
-		build.NewRouter(dockerfile.NewBuildManager(d)),
+		build.NewRouter(d),
```

Comment:
This is probably the sweetest line for me. We're at all the routers taking a straight `backend`, with no ties. It's nice.